docs/topics/braintree.rst

```diff
@@ -3,4 +3,23 @@
 Braintree
 #########
 
-More to come.
+Generate a token
+----------------
+
+Calls braintree `ClientToken.generate <https://developers.braintreepayments.com/javascript+python/reference/request/client-token/generate>`_:
+
+.. http:post:: /braintree/token/generate/
+
+    **Request**
```

Comment:
You get that information by querying the Customer API in Braintree. For our purposes we'll be storing it in solitude along with the buyer, so it will come back when you query the buyer.